src/App.jsx

```diff
@@ -1,107 +1,25 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
+
+const preparedPosts = postsFromServer.map((post) => {
+  const user = usersFromServer.find(u => post.userId === u.id);
```

Comment:
'u' is not good idea for naming, names of your variables should not confuse your colleagues